pantheon-systems/WordPress
# Update to WordPress 4.4.1.

For more information, see https://wordpress.org/news/2016/01/wordpress-4-4-1-security-and-maintenance-release/

Instructions to test:

- Create a new WordPress site on Pantheon.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add wp git@github.com:pantheon-systems/WordPress
  - git fetch wp
  - git merge wp/wp-4.4.1
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process.
- Confirm that the checksums of all of the core files match:
  - terminus --site=YOURWORDPRESSSITE wp core verify-checksums
  - Should say: "Success: WordPress install verifies against checksums."
